### ISSUE TITLE
Neo 0.5.0: error in reading .abf file header

### ISSUE THREAD

**Issue by lbologna** [NeuralEnsemble/python-neo#302]
Hi,

I am trying to read the header of .abf files with Neo 0.5.0 but I get the following error:

assert goodstart!=-1, 'This file do not contain clampex, axoscope or clampfit in the header'

Nonetheless, I could read the header with Neo 0.4.1.

I believe the file has been converted from .dat to .abf via pClamp tools.

I can send you privately the .abf if needed.

Thanks for your help.
Kind regards,

Luca Bologna

**Comment by apdavison** [comment]
Hi Luca,

Yes, please, could you send me the file.

Andrew

**Comment by lbologna** [comment]
Hi Andrew,

Just sent you an example file.

Thanks.

Luca

**Comment by apdavison** [comment]
Hi Luca,

I have a possible fix. Please could you try it, either by installing Neo from the "issue302" branch of https://github.com/apdavison/python-neo/, or manually editing your current Neo installation according to 56c5380.

Thanks,

Andrew

**Comment by lbologna** [comment]
Hi Andrew,

I edited the code on my installation and everything worked well.

Thanks!

Luca

**Comment by samuelgarcia** [comment]
Closed by #303

**Comment by mqclark** [comment]
I think I'm having the same issue as Luca. 

File "/Users/matt/anaconda3/lib/python3.6/site-packages/neo/rawio/axonrawio.py", line 458, in parse_axon_soup
    'This file does not contain clampex, axoscope or clampfit in the header'

AssertionError: This file does not contain clampex, axoscope or clampfit in the header

I checked the solution previosly stated, however it appears in the latest version I downloaded that that had been fixed.

Here is the .abf in the hopes it can be looked at. 
https://drive.google.com/open?id=1CAXLYRNwLsg5vXIH8SABKod1TCrvwJK0

Cheers,
Matthew Clark
